tests/installation/partitioning.pm

```diff
@@ -22,15 +22,18 @@ use partition_setup qw(%partition_roles is_storage_ng_newui);
 
 sub run {
     assert_screen 'partitioning-edit-proposal-button', 40;
-    if (check_var('PARTITION_EDIT', 'ext4_btrfs')) {
+    # We need to test on different filesystems on root partition
+    # and do not care the filesystem on home partition.
+    my $my_filesys = get_var('PARTITION_EDIT');
```

Comment:
If don't set 'PARTITION_EDIT', the $my_filesys will get undefined.

Reply:
Not set then we won't run the following code flow.

Reply:
undefine should not be '1', so the condition should work. :)